src/backend/preload.ts

```diff
@@ -2,12 +2,14 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { contextBridge } from 'electron'
 import api from './api'
+import { isVulkanSupported } from './utils/graphics/vulkan'
 
 contextBridge.exposeInMainWorld('api', api)
 contextBridge.exposeInMainWorld(
   'isSteamDeckGameMode',
   process.env.XDG_CURRENT_DESKTOP === 'gamescope'
 )
+contextBridge.exposeInMainWorld('isVulkanSupported', isVulkanSupported())
```

Comment:
this doesn't work because the constants are not initialized yet, I have no clue how to fix it